tests/contrib/tornado/web/compat.py

```diff
@@ -24,12 +25,16 @@ def __init__(self, *args, **kwargs):
             super(ThreadPoolExecutor, self).__init__()
 
 
-def sleep(duration):
-    """
-    Compatibility helper that return a Future() that can be yielded.
-    This is used because Tornado 4.0 doesn't have a ``gen.sleep()``
-    function, that we require to test the ``TracerStackContext``.
-    """
-    f = Future()
-    IOLoop.current().call_later(duration, lambda: f.set_result(None))
-    return f
+if hasattr(tornado.gen, "sleep"):
```

Comment:
```suggestion
if hasattr(tornado.gen, 'sleep'):
```

Reply:
This was based on `master` somehow, so I missed it. Damn.